repour/scm/git_provider.py

```diff
@@ -105,7 +105,7 @@ def push_with_tags(dir, branch, remote="origin", tryAtomic=True):
         def do(atomic):
             yield from expect_ok(
                 cmd=["git", "push"] + (["--atomic"] if atomic else []) + ["--follow-tags", remote, branch],
-                desc="Could not" + (" atomic" if atomic else "") + " push tag+branch with git",
+                desc="Could not" + (" atomic" if atomic else "") + " push tag+branch with git. Make sure you have the right permissions!",
```

Comment:
Something like:

"Could not push tag+branch with git. Make sure the pnc-user has push permissions to this gerrit repository"
?